### PR TITLE
Update JDK8 build instructions using VS2013 for Windows

### LINKED ISSUE #11477: Update Java 8 Windows build instructions for VS2013
When switching to VS2013 for Windows Java 8 builds, the build instructions were never updated.
https://github.com/eclipse/openj9/blob/master/doc/build-instructions/Build_Instructions_V8.md

Building with VS2010 no longer works as the code isn't compatible, the instructions need to be updated.

### PR DESCRIPTION
**Pull request by jdekonin**
* [skip ci]
* Resolves #11477

Signed-off-by: Joe deKoning <joe_dekoning@ca.ibm.com>

### CODE REVIEW

**Comment by jdekonin** [comment]
@pshipton for review.

**Review by pshipton — APPROVED** [review]

File: doc/build-instructions/Build_Instructions_V8.md [hunk 1]
```diff
@@ -378,12 +377,12 @@ set INCLUDE=C:\Program Files\Debugging Tools for Windows (x64)\sdk\inc;%INCLUDE%
 set LIB=C:\Program Files\Debugging Tools for Windows (x64)\sdk\lib;%LIB%
 ```
 
-Not all of the shared libraries that are included with Visual Studio 2010 are registered during installation.
-In particular, the `msdia100.dll` libraries must be registered manually.
+Not all of the shared libraries that are included with Visual Studio 2013 are registered during installation.
+In particular, the `msdia120.dll` libraries must be registered manually.
 To do so, execute the following from a command prompt:
```
Comment (pshipton) [inline doc/build-instructions/Build_Instructions_V8.md:382]: As per https://github.com/eclipse/openj9/issues/11477#issuecomment-748214560 the following needs to be done as administrator, which can be added here, and for 11, and 15 as well.

File: doc/build-instructions/Build_Instructions_V8.md [hunk 2]
```diff
@@ -405,25 +404,6 @@ tar -xzf freemarker.tgz freemarker-2.3.8/lib/freemarker.jar --strip=2
 tar --one-top-level=/cygdrive/c/temp/freetype --strip-components=1 -xzf freetype-2.5.3.tar.gz
 ```
 
-Note:
-In order to build Windows 32-bit, Please enure the freetype version supports win 32-bit application (e.g. freetype-2.4.7 which includes the "win32" folder)
-
-- To build the Freetype dynamic and static libraries, open the Visual Studio Command Prompt (VS2010) (see C:\ProgramData\Microsoft\Windows\Start Menu\Programs\Microsoft Visual Studio 2010\Visual Studio Tools) and run:
```
Comment (harpreetbamrah) [inline doc/build-instructions/Build_Instructions_V8.md:411]: We still need to build freetype libraries, but from Developer Command Prompt for VS2013 (C:\ProgramData\Microsoft\Windows\Start Menu\Programs\Visual Studio 2013\Visual Studio Tools) and running respective commands for Win 64-bit and 32-bit. Just the note for freetype to include the "win32" folder is no longer required.

Hope this helps!

Comment (jdekonin) [reply doc/build-instructions/Build_Instructions_V8.md:411]: I believe that might be the case if the `configure` options are not modified.  Note that I made a change to those as well.

`--with-freetype-include=/cygdrive/c/temp/freetype/include --with-freetype-lib=/cygdrive/c/temp/freetype/lib64` becomes `--with-freetype-src=/cygdrive/c/temp/freetype`  With this in place I get the following output from configure :

```
configure: Trying to compile freetype sources with PlatformToolset=v120 to /cygdrive/c/temp/freetype/lib64/ ...
configure: Compiling freetype sources succeeded! (see freetype.log for build results)
configure: Found freetype include files at /cygdrive/c/temp/freetype/include using --with-freetype-src
configure: Rewriting POTENTIAL_FREETYPE_INCLUDE_PATH to "/cygdrive/c/temp/freetype/include"
configure: Rewriting POTENTIAL_FREETYPE_LIB_PATH to "/cygdrive/c/temp/freetype/lib64/"
checking for freetype includes... /cygdrive/c/temp/freetype/include
checking for freetype libraries... /cygdrive/c/temp/freetype/lib64/
checking if we can compile and link with freetype... yes
checking if we should bundle freetype... yes
```

File: doc/build-instructions/Build_Instructions_V8.md (hunk 1 above)
Comment (pshipton) [reply doc/build-instructions/Build_Instructions_V8.md:382]: @jdekonin if you can update this, I think the PR will be ready for merge.

Comment (jdekonin) [reply doc/build-instructions/Build_Instructions_V8.md:382]: Thank you for pointing that out @pshipton.  PR updated.

Comment (pshipton) [reply doc/build-instructions/Build_Instructions_V8.md:382]: I don't think `Administrator` should be capitalized. Can you please add this bit to the jdk11 and 15 file as well.

Comment (pshipton) [reply doc/build-instructions/Build_Instructions_V8.md:382]: Seems both Administrator and administrator (like `Run as administrator`) are used.

File: doc/build-instructions/Build_Instructions_V8.md (hunk 2 above)
Comment (harpreetbamrah) [reply doc/build-instructions/Build_Instructions_V8.md:411]: ok, that makes sense.

I just noticed a typo - adminstrative privledges.

Comment (jdekonin) [reply doc/build-instructions/Build_Instructions_V8.md:411]: Updated.

**Review by pshipton — APPROVED** [review]

**Comment by harpreetbamrah** [comment]
I somehow realized that if we have multiple versions of Visual Studio installed, we need to include --with-toolchain-version=2013 option in the configure command.

**Comment by pshipton** [comment]
@harpreetbamrah pls consider opening a PR if you want to make additional changes to the instructions.

**Pull request merged**